src/net.h

```diff
@@ -624,14 +649,19 @@ class CNetMessage {
     CDataStream vRecv;              // received message data
     unsigned int nDataPos;
 
-    int64_t nTime;                  // time (in microseconds) of message receipt.
+    TransportDeserializer(const CMessageHeader::MessageStartChars& pchMessageStartIn, int nTypeIn, int nVersionIn) : hdrbuf(nTypeIn, nVersionIn), hdr(pchMessageStartIn), vRecv(nTypeIn, nVersionIn) {
+        Reset();
+    }
 
-    CNetMessage(const CMessageHeader::MessageStartChars& pchMessageStartIn, int nTypeIn, int nVersionIn) : hdrbuf(nTypeIn, nVersionIn), hdr(pchMessageStartIn), vRecv(nTypeIn, nVersionIn) {
+    void Reset() {
```

Comment:
In commit "Refactor: split network transport deserializing from message container" (6294ecdb8bb4eb7049a18c721ee8cb4a53d80a06)

Reset method seems to reset every member except `hdr`, leaving it untouched. I guess this is because `CMessageHeader` doesn't have a clear method or easily callable constructor, but it would be good to have a comment saying `hdr` is intentionally skipped, because this looks like a bug, and could potentially cause bugs in the future.